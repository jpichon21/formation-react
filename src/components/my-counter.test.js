/* globals test, expect */
import React from 'react'
import renderer from 'react-test-renderer'
import Counter from './counter';

test('Navbar snapshot', () => {
 const component = renderer.create(<Counter/>)

 let tree = component.toJSON()

 // Vérifier que le composant n'a pas changé depuis le dernier
 // snapshot.
 // Voir https://facebook.github.io/jest/docs/en/snapshot-testing.html
 // pour plus d'informations
 expect(tree).toMatchSnapshot()

 // L'API expect() de Jest est disponible à l'adresse
 // https://facebook.github.io/jest/docs/en/expect.html

 // Il est possible d'effectuer des vérifications plus avancées
 // grâce au projet Enzyme (vérification du DOM, etc)
 // Voir http://airbnb.io/enzyme/ et
 // https://facebook.github.io/jest/docs/en/tutorial-react.html#dom-testing
})