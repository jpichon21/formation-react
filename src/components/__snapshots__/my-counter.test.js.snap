// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar snapshot 1`] = `
<div>
  Count: 
  <span>
    0
  </span>
  Â 
  <button
    onClick={[Function]}
  >
    +
  </button>
  <button
    onClick={[Function]}
  >
    -
  </button>
</div>
`;
